govcd/catalog_test.go

```diff
@@ -284,6 +284,15 @@ func (vcd *TestVCD) Test_UploadOvf_cleaned_extracted_files(check *C) {
 
 }
 
+// Tests System function UploadOvf by creating catalog and
+// checking if provided standard ovf file uploaded.
+func (vcd *TestVCD) Test_UploadOvfFile(check *C) {
+	fmt.Printf("Running: %s\n", check.TestName())
+
+	skipWhenOvaPathMissing(vcd, check)
```

Comment:
This check is wrong. We should check that ovfPath is filled, but we are only checking OVA path. We need an explicit check for ovf path here

Reply:
Thank you. I've fixed the issue in my another branch which planed to be pull after this. Allow me to get the fixed included in this pull right now. See the new commit below.